data/strings/types_strings.txt

```diff
@@ -23312,7 +23069,8 @@
     zh-Hant = 家電店
 
   [type.shop.art]
-    en = Arts Shop
+    comment = maybe change to Art Gallery for en-US when supported
```

Comment:
Why we can't use now en = Art Gallery for the US, and en-GB for Artwork Shop?

The same for all other Shop/Store? E.g. use Store for all en, and use Shop for en-GB?

Reply:
I guess the question is what people in CA, AU, NZ, IE call these places.  I suspect US/CA is Gallery and the rest is shop.   I don't like "Arts Shop" as while that might be right in GB, it will confuse US readers.  But "Artwork Shop", while it will seem odd to an en_US speaker, will likely be understood.

Reply:
Artwork Shop is much better, I'll change to that 👍 the issue is trying to make it clear that the art is primarily displayed for sale, as opposed to a national/museum art gallery, where the art is displayed as an attraction, and not for sale.